source/components/molecules/ScreenWrapper.js

```diff
@@ -14,7 +15,13 @@ const ScreenWrapper = props => {
 
   return (
     <ThemeProvider theme={theme}>
-      <Container style={style}>{children}</Container>
+      <Container
+        behavior={Platform.OS == 'ios' ? 'padding' : 'height'}
```

Comment:
Platform is not defined and ===. May be run Yarn run lint to see the errors

Reply:
Fixed.